### PR TITLE
[Lib] add OpenDefaultApp event model in Solutions

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
Lib part of #2338 

### Changes
Add OpenDefaultApp event model.

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

#### Bots
- [ ] I have validated that new and updated responses use appropriate [Speak](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-text-to-speech?view=azure-bot-service-3.0) and [InputHint](https://docs.microsoft.com/en-us/azure/bot-service/dotnet/bot-builder-dotnet-add-input-hints?view=azure-bot-service-3.0) properties to ensure a high-quality speech-first experience
- [ ] I have replicated language model changes across the English, French, Italian, German, Spanish, and Chinese `.lu` files and validated that deployment is successful

#### Deployment Scripts
- [ ] I have replicated my changes in the **Virtual Assistant Template** and **Sample** projects
- [ ] I have replicated my changes in the **Skill Template** and **Sample** projects

### CODE REVIEW

File: lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Models/OpenDefaultApp.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Bot.Builder.Solutions.Models
+{
+    public class OpenDefaultApp
+    {
+        public string MeetingUri { get; set; }
+
+        public string TelephoneUri { get; set; }
```
Comment (ryanisgrig) [inline lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Models/OpenDefaultApp.cs:10]: Add MapsUri

**Comment by ryanisgrig** [comment]
Let's wait for 0.7 for this one - I want to make sure we cover all the Uri scenarios

File: lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Models/OpenDefaultApp.cs (hunk 1 above)
Comment (ryanisgrig) [reply lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Models/OpenDefaultApp.cs:10]: Add MusicUri

**Review by darrenj — APPROVED** [review]

File: lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Models/OpenDefaultApp.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+        public string MapUri { get; set; }
```
Comment (ryanisgrig) [inline lib/csharp/microsoft.bot.builder.solutions/microsoft.bot.builder.solutions/Models/OpenDefaultApp.cs]: Rename to MapsUri

**Review by ryanisgrig — CHANGES_REQUESTED** [review]

**Review by ryanisgrig — APPROVED** [review]

**Pull request merged**